src/components/clickable-image/ClickableImage.styles.ts

```diff
@@ -0,0 +1,29 @@
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  imageButton: {
+    width: '88px',
+    height: '88px',
+    '& div': {
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      position: 'absolute',
+      top: '50%',
+      left: '50%',
+      transform: 'translate(-50%, -50%)',
```

Comment:
I think, you don`t need to center it by absolute if you are using transform, try this without absolute position

Reply:
> I think, you don`t need to center it by absolute if you are using transform, try this without absolute position

`position: 'absolute'` used here not only to change position, but also for stretching relative to the parent block. Also that's fixing the problems like pushing parent block slightly to the left, so i suppose it would be better to leave it like this 😅

![зображення](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/45912519/a3cbbcb9-21ed-4ad0-a1b9-e7c43f7f8b4d)

Reply:
if you want to stretch the child block, try flex property

Reply:
I need that absolute positioning, so I just removed the transform property 😅